docs/Developer's ReadMe.md

```diff
@@ -123,25 +123,31 @@ Set up your Theia workspace for development purposes.
 
 **Follow these steps**:
 
-1. Build a VSIX file with your changes.
+1. Make sure you have the following installed:
 
-2. Build and run the Theia browser example, using [Setting up your Theia workspace](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md#setting-up-your-theia-workspace).
+   - [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/)
+   - [Mozilla Geckodriver](https://github.com/mozilla/geckodriver/releases) (and add it to your PATH)
+   - [ChromeDriver](https://chromedriver.chromium.org/) (and add it to your PATH)
+   - Selenium webdriver with `npm i -g selenium-webdriver`
 
-   **Note**: Ensure that your latest VSIX file is in the `plugins` folder.
+2. Build a VSIX file with your changes.
 
-3. Open a web browser and navigate to <http://localhost:3000> to verify your setup.
+3. Build and run the Theia browser example, using [Setting up your Theia workspace](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md#setting-up-your-theia-workspace).
+
+   - Ensure that your latest VSIX file is in the `plugins` folder.
+   - Theia must be **running** while you execute the tests!
+
+4. Open a web browser and navigate to <http://localhost:3000> to verify your setup.
```

Comment:
If the point of this step is to verity the setup, I'd word it as follows: 
"(Optional) To verify your setup, open a browser and navigate to <http://localhost:3000>".

These verification steps are usually optional hence the `(optional)` marker